README.md

```diff
@@ -81,17 +96,35 @@ Configure settings in the /etc/cockpit/cockpit.conf file.  See [`man cockpit.con
     cockpit_port: 9090
 Cockpit runs on port 9090 by default. You can change the port with this option.
 
-Note that the default SELinux policy does not allow Cockpit to listen to anything else than port 9090, so you need to allow that first, with e.g.
+    cockpit_manage_firewall: false
```

Comment:
We really should be using `true` and `false` everywhere - https://github.com/redhat-cop/automation-good-practices/blob/main/coding_style/README.adoc#yaml-and-jinja2-syntax "Use true and false for boolean values in playbooks."

but yes, it will look strange here if we are not consistent, but we may want to convert everything in the future - note that some of the other roles have already done this - https://github.com/linux-system-roles/ha_cluster/pull/60

Reply:
Ok. `yes/no` for cockpit for now. Let me update README and tests.
I assume the change is not needed in the files in tasks and defaults... Right?

Reply:
> Ok. `yes/no` for cockpit for now. Let me update README and tests. I assume the change is not needed in the files in tasks and defaults... Right?

Please use `yes` and `no` in all of the code you are adding.